rules/github-scrum-workflow/rule.md

```diff
@@ -0,0 +1,84 @@
+---
+type: rule
+archivedreason:
+title: How to set up GitHub Scrum workflow
+guid: e63da194-c5d2-4830-9106-d164aafcddb5
+uri: do-you-know-the-how-to-set-up-github-workflow
+created: 2021-06-21T06:44:34.0000000Z
+authors:
+  - title: Tom Bui
+    url: https://www.ssw.com.au/people/tom-bui
+related: []
+
+---
+### How to set up GitHub Scrum workflow
+
+There are several ways of turning your GitHub repository into a simple but powerful agile work environment, the following is the recommended:
+
+<!--endintro-->
+
+Based on [GitHub scrum workflow](https://github.com/jvandemo/github-scrum-workflow)
+
+#### How it works
+
+- Product Backlog Items (PBIs) are reported as **issues**
+- Points and meta data are assigned to items as **labels**
+- **milestones** are used to group issues in Sprints
+
+#### 1. Create issues as backlog items
+
+To create a new backlog item, just create a new issue.
+
+Once a new issue has been created, assign it the right labels and/or assign it to a Sprint (milestone).
+
+Issues allow you to have a conversation about the item and even allow you to create task lists inside the issue using [GitHub's markdown](https://guides.github.com/features/mastering-markdown/).
+
+#### 2. Add labels to issues
+
+Add the following labels to your repository:
+
+### Estimates
+
+`estimate` labels allow you to set estimates in your backlog:
+e.g.:
+| Label        | Time Estimate |
+| -------------|:-------------:|
+| `estimate: 1` | trivial |
+| `estimate: 2` | 1 hour |
+| `estimate: 3` | 2 hours |
+| `estimate: 5` | 4 hours |
+| `estimate: 8` | 1 day |
+| `estimate: 13` | 1.5 days |
+| `estimate: 21` | 3 days |
+| `estimate: 34` | 1 week |
+
+Your actual estimates should align with https://www.ssw.com.au/rules/estimating-do-you-know-how-to-size-user-stories-effectively
+
+### Types
+
+`type` labels allow you to easily filter items (issues) in the dashboard e.g.:
+
+| Label | Type |
+| ------| :----|
+| `type:bug`| bug |
+| `type:chore`| chore, maintenance work |
+| `type:feature`| new feature |
+| `type:infrastructure` | infrastructure related |
+| `type:performance` | performance related |
+| `type:refactor` | refactor |
+| `type:test` | test related |
+
+### Other
+
+You can define and assign custom labels that you need within your workflow or organization.
+
+#### 3. Define Sprints as milestones
+
+You can create a milestone for every Sprint and add items (issues) from the backlog to a milestone.
```

Comment:
Should we always call them issues, is there a reason to use the terminology "item" at all? May be valid as is, just posing the question.

Reply:
I think the term "items" refers to "Product Backlog Items" which is a more standardized term in Scrum than 'issues'

Reply:
If that's the case I think we should use the term "PBIs" and on the first one you can put PBIs (issues) and then remove other references to "issues". What do you think?

Reply:
Hey @pierssinclairssw , I think we should leave it as it is because the first point already states that issues will be used as PBIs. And based on the context we use the term "issue" when we mention about GitHub and "PBIs" when it is about the context of Scrum

<img width="445" alt="image" src="https://user-images.githubusercontent.com/67776356/122842711-3d513800-d341-11eb-8f17-124eb4bcf1c6.png">

Reply:
What I mean to say is we replace the word "items" with "PBIs" not the word "issues".

Right now we have 3 different words. I think PBIs and items are the same thing so in my opinion, it would be better to consolidate that term into just PBIs. We also use PBIs in other places around sugarlearning, rules etc in my experience.

My main concern is that when I first read the rule I was confused by the changes in terminology as I was reading.

Reply:
Thanks @pierssinclairssw,  I do agree
> replace the word "items" with "PBIs" not the word "issues"

Done